common/rfb/CSecurityTLS.cxx

```diff
@@ -217,7 +217,11 @@ void CSecurityTLS::setParam()
   if (anon)
     strcat(prio, kx_anon_priority);
 
+#if GNUTLS_VERSION_NUMBER >= 0x030603
+  ret = gnutls_set_default_priority_append(session, prio, &err, 0);
+#else
   ret = gnutls_priority_set_direct(session, prio, &err);
+#endif
```

Comment:
This ended up a bit too far back. We still need to change the default to `""`, and call `gnutls_priority_set_direct()` in every case where it is not the empty string.

It is the specific case of anonymous mode and empty priority string that needs the latest round of special handling.

Reply:
Ok, I hope I understood it correctly and didn't lost in all those possible scenarios.